apps/arweave/src/ar_peers.erl

```diff
@@ -375,13 +375,20 @@ resolve_and_cache_peer(RawPeer, Type) ->
 %%%===================================================================
 
 init([]) ->
-	%% Trap exit to avoid corrupting any open files on quit.
-	process_flag(trap_exit, true),
-	ok = ar_events:subscribe(block),
-	load_peers(),
-	gen_server:cast(?MODULE, rank_peers),
-	gen_server:cast(?MODULE, ping_peers),
-	timer:apply_interval(?GET_MORE_PEERS_FREQUENCY_MS, ?MODULE, discover_peers, []),
+	{ok, Config} = application:get_env(arweave, config),
+	case Config#config.verify of
```

Comment:
Why do we turn it off?

Reply:
Mostly for performance reasons. Pinging the peers seems to be one of the slower parts during node launch. That said... not completely sure this is the right approach. I slacked you with more context.